src/cli/llm/wizard.rs

```diff
@@ -5,20 +5,19 @@ use genai::resolver::AuthResolver;
 use genai::Client;
 
 use super::Result;
-use crate::cli::llm::model::Model;
 
 #[derive(Setters, Clone)]
 pub struct Wizard<Q, A> {
     client: Client,
-    model: Model,
+    model: String,
```

Comment:
We need a type-safe way to configure the LLMs. Please add the `model` back. I think you will need to convert the model into an ENUM.